src/ejercicios.js

```diff
@@ -9,9 +9,17 @@ export default class Ejercicios {
     }
 
     conId(id) {
-        let ejercicio = this.ejercicios.find((unEjercicio) => { return unEjercicio.props.id === id });
+        let ejercicio = this.buscarEjercicio(id);
```

Comment:
aca podria ser `let ejercicio = this.buscarEjercicio(id) || this.buscarEjercicio(1)`, nos evitariamos el if de abajo, de todas formas entiendo si te parece mejor dejar el if.. es un poco mas explicito que el `||`

Reply:
El if queda un poco mas explicito y me gustaria que quede como un caso para revisar en algun futuro. Tal vez pensar con los chicos que les gustaria que pase por default si algun usuario llega a entrar a este caso. Tener como default al ejercicio 1 puede no ser lo mejor tampoco... Lo dejo asi y que duela verlo